package.json

```diff
@@ -116,6 +116,7 @@
     "gh-pages": "^2.0.1",
     "glob": "^7.1.3",
     "idb-keyval": "2.2.0",
+    "intl-messageformat-parser": "^1.8.1",
```

Comment:
¯\\_(ツ)_/¯

Reply:
lol I thought we just removed this as unnecessary?

Reply:
> lol I thought we just removed this as unnecessary?

ha, yeah. I mean, it could use `new MessageFormat(locale).getAst()` but that seemed silly, so back it comes. At least it's no change for what's in `node_modules/` :)